app/models/question.rb

```diff
@@ -14,6 +14,7 @@
 class Question < ActiveRecord::Base
   belongs_to :user
   has_many :answers
+  has_one :approved_answer, class_name: "Answer", foreign_key: "id", primary_key: "approved_answer_id"
```

Comment:
Line is too long. [102/100]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.